rna_transcription.py

```diff
@@ -1,11 +1,10 @@
-def to_rna(dna_strand):
-    rna = ''
-    for x in dna_strand:
-        rna = rna + {
+dna_to_rna = {
             'G': 'C',
             'C': 'G',
             'T': 'A',
             'A': 'U'
-        }[x]
-    return rna
-print(to_rna('ACGTGGTCTTAA'))
+        }
+
+
+def to_rna(dna_strand):
+    return ''.join([dna_to_rna[x] for x in dna_strand])
```

Comment:
@zarniak 💯 

And... how would that look like with a built-in `map` function? :trollface:

Reply:
@virtuallight like this :)
    return ''.join(list(map(dna_to_rna.get, dna_strand)))
dont know which is better - #1 is simpler to encode i think, #2 is simpler but need knowledge/expierence/google to encode